Inc/HALAL/Services/CORDIC/CORDIC.hpp

```diff
@@ -66,13 +66,13 @@ class RotationComputer{
 	 * If the angle or the out parameter has less size than the size parameter, it will throw an exception
 	 * If one or both angle and out parameters have more size than size, it will just calculate until size and leave the rest as it is
 	 */
-	static void cos(int32_t *angle,int32_t *out,int size);
+	static void cos(int32_t *angle,int32_t *out,int32_t size);
 
 
 	/*
```

Comment:
Para doxygen hay que poner dos asteriscos al principio de cada comentario. 
```suggestion
	/**
```

Reply:
Ese cambio deberia hacerlo en el propio CORDIC, porque es de esa clase. Supongo que si lo hago aquí no debería haber problema ya que vamos a sacar el release así que todo se juntara, pero podriamos hacer una pull request para cosas del doxygen y documentacion.